pkg/driver/controller.go

```diff
@@ -354,27 +354,20 @@ func (d *controllerService) ControllerPublishVolume(ctx context.Context, req *cs
 	volumeID := req.GetVolumeId()
 	nodeID := req.GetNodeId()
 
-	if !d.cloud.IsExistInstance(ctx, nodeID) {
-		return nil, status.Errorf(codes.NotFound, "Instance %q not found", nodeID)
-	}
-	disk, err := d.cloud.GetDiskByID(ctx, volumeID)
```

Comment:
Why is it safe to remove ``ec2.DescribeVolume`` call?

Reply:
It should probably be included in the PR description, but by my understanding this check is superfluous and removed for efficiency.

Reply:
Why was  this check added before?